client/assets/js/admin-controller.js

```diff
@@ -193,15 +199,21 @@ angular.module('portal').controller('adminController', function ($route, $scope,
                 showConfirmButton: false
             });
         });
+        
     };
 
 
+
+    // sets user's isAdmin param to false
     $scope.removeAdmin = function() {
 
         $http.get('/api/user', {
-            params : { 'email': $scope.changeAdminEmail }
+            params : { 
+                'email': $scope.changeAdminEmail, 
+                'curEmail': $scope.curUser.email,
```

Comment:
I believe `curEmail` and `curAdmin` are unnecessary here, see my comment in `user.js`.